metagraph/src/graph/representation/succinct/boss_chunk_construct.cpp

```diff
@@ -404,6 +404,27 @@ void skip_same_suffix(const KMER &el, Decoder &decoder, size_t suf) {
     }
 }
 
+int check_fd_and_adjust_threads(size_t num_threads, size_t fd_per_thread) {
```

Comment:
why not move this to `common`? This could potentially be useful in other parts of the code.

Reply:
makes sense